test.py

```diff
@@ -641,7 +641,7 @@ def write_junit_failure_report(self, xml_res: ET.Element) -> None:
 class UnitTest(Test):
     standard_args = shlex.split("--overprovisioned --unsafe-bypass-fsync 1 "
                                 "--kernel-page-cache 1 "
-                                "--blocked-reactor-notify-ms 2000000 "
+                                "--blocked-reactor-notify-ms 2000000 --collectd 0 "
```

Comment:
Please fold it into the patch the changed it, we try to avoid going back and forth within a series.

Reply:
folded and pushed.